algebird-test/src/test/scala/com/twitter/algebird/statistics/GaussianDistributionMonoidTests.scala

```diff
@@ -0,0 +1,21 @@
+package com.twitter.algebird.statistics
+
+import com.twitter.algebird.CheckProperties
+import org.scalacheck.{ Arbitrary, Gen }
+import org.scalatest._
+
+class GaussianDistributionMonoidTests extends CheckProperties {
+  import com.twitter.algebird.BaseProperties._
+
+  implicit val gaussianMonoid = GaussianDistributionMonoid
```

Comment:
shouldn't need this. Since it is implicit in the companion object, scala should find it.